conf/machine/hikey.conf

```diff
@@ -37,3 +42,4 @@ CMDLINE = "console=ttyAMA0,115200n8 root=/dev/mmcblk1p2 rootwait ro maxcpus=8 ea
 IMAGE_FSTYPES_append = " ext4"
 
 EXTRA_IMAGEDEPENDS = "edk2 l-loader"
+PREFERRED_VERSION_mesa = "11.%"
```

Comment:
mesa 11 is not in OE core, yet. we will wait for it to be merged before we can merge this change.